.github/workflows/run-tests.yml

```diff
@@ -1,9 +1,10 @@
 name: Full Tests
 on:
-  schedule:
-    - cron: "0 0 * * *"  # daily
   pull_request:
   workflow_dispatch:
+  workflow_run:
+    workflows: [update-testing-data]
+    types: [completed]
```

Comment:
If going this route, you'll probably want to add the 'on success' logic to allow early exit (otherwise will proceed to run tests and fail when it doesn't find a certain file/folder); I wonder if `success` is an allowed type for the trigger itself?

Reply:
Yeah that seems reasonable.  Apparently `on: workflow_run` only works on the default branch ([see Issue](https://github.com/orgs/community/discussions/25288)), so I can't test it before merging? Kind of annoying.

Reply:
> Yeah that seems reasonable. Apparently on: workflow_run only works on the default branch ([see Issue](https://github.com/orgs/community/discussions/25288)), so I can't test it before merging? Kind of annoying.

This kind of thing is typical of GitHub Action devops stuff - it's the one situation where committing to main to debug stuff is OK IMO

I'll approve and merge this and let you continue messing with it